tests/datasource/data_connector/test_runtime_data_connector.py

```diff
@@ -27,7 +27,7 @@
 
 
 @pytest.fixture
-def basic_datasource_with_assets(tmp_path_factory):
+def basic_datasource_with_assets():
```

Comment:
This doesn't actually use the filesystem so updated this fixture and the test markers from filesystem to unit.

Reply:
Great change. Thanks